tendrl/node_agent/tests/test_disk.py

```diff
@@ -0,0 +1,92 @@
+from tendrl.node_agent.persistence.disk import Disk
+
+
+class Test_Disk(object):
```

Comment:
Avoid underscores in class names and rather use camelCase with first letter uppercase